src/CSnakes.Runtime/EnvironmentManagement/CondaEnvironmentManagement.cs

```diff
@@ -0,0 +1,37 @@
+﻿using CSnakes.Runtime.Locators;
+using Microsoft.Extensions.Logging;
+
+namespace CSnakes.Runtime.EnvironmentManagement;
+internal class CondaEnvironmentManagement(string name, bool ensureExists, CondaLocator conda, string? environmentSpecPath) : IEnvironmentManagement
```

Comment:
Is there a need for `environmentSpecPath` if it's not used?

Reply:
yes, because it'll be used in future. `ensureExists` isn't implemented but it'll need that parameter and I don't want to change the API when it is implemented. For now you get a not supported error